### PR TITLE
Routes to page

### PR DESCRIPTION
**Pull request by ykostov**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
-  get 'pages/home'
+  # get 'pages/home'
   get 'pages/help'
```
Comment (rmilushev) [inline config/routes.rb:3]: We have this on row 11, please, remove it.

File: config/routes.rb [hunk 2]
```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
-  get 'pages/home'
+  # get 'pages/home'
```
Comment (rmilushev) [inline config/routes.rb]: Please, do not remove this line, it is needed for `root to:` routine on line 11.

File: app/views/pages/tour.html.erb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<div class="navbar">
```
Comment (rmilushev) [inline app/views/pages/tour.html.erb]: We can extract the common `navbar` to a partial file with a name `_navbar.html` and render it on `layout/application.html.erb` template. Use for as a reference our other projects, please.

File: config/routes.rb [hunk 4]
```diff
@@ -1,6 +1,12 @@
...
   get 'pages/help'
+  get 'pages/about'
+  get '', to: "pages#home"
```
Comment (rmilushev) [inline config/routes.rb]: `get` with empty string is not ok :). We have `root` path on row 11, please, remove this.

**Review by rmilushev — CHANGES_REQUESTED** [review]

**Review by N-Patarov — APPROVED** [review]

File: app/views/layouts/_navbar.html [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<div class="navbar">
+  <a href="/">Home Page </a>
```
Comment (rmilushev) [inline app/views/layouts/_navbar.html]: One last touch, please: in Rails, we use `route helpers` in the form:
```
<%= link_to 'Home page', root_path %>
```
Try to refactor the links here to the Rails way, and you will see how easy to use they are. Hint: for example, link to the `ranklist` is: 
```
<%= link_to 'Ranklist', ranklist_path %>
```

**Review by rmilushev — CHANGES_REQUESTED** [review]

**Review by rmilushev — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**